06.bowling_object/game.rb

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+require './shot'
+require './frame'
+
+class Game
+  def initialize(score)
+    @frames = parse_frames(score)
+  end
+
+  def parse_frames(score)
+    frames_mark = score.split(',')
```

Comment:
```suggestion
    marks = score.split(',')
```

複数形になるのはmarkの方だと思います。
あと、「frameのmark」以外がないなら「frameの」は省略してもよさそうです

Reply:
marksに変更しました。